CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+This document is a guide for contributions to the WGPU project.
+
+## Welcome!
+
+First of all, welcome to the WGPU community! 👋 We're glad you want to
+contribute. If you are unfamiliar with the WGPU community as an organization, we
+recommend you read [`GOVERNANCE.md`] for an overview of how it works.
+
+[`GOVERNANCE.md`]: ./GOVERNANCE.md
+
+## Talking to other humans in the WGPU project
+
+The WGPU project has multiple ways to engage:
+
+- [GitHub discussions]: TODO: not sure when to use this over Matrix and issues.
+  TBH.
```

Comment:
issue: This does not seem to receive significant attention from current maintainership. If we officially mention this here, we should say that, at best, GitHub discussions are a community experiment.

Reply:
Let's leave it out for now. We can always add it later if we start properly monitoring/using it.